clang/test/CodeGen/builtins-nvptx-native-half-type.c

```diff
@@ -0,0 +1,174 @@
+// REQUIRES: nvptx-registered-target
```

Comment:
Could you please add a comment what test is doing?

Reply:
This change (everything in clang folder) are already merged upstream. I just added them to this PR as they are required to build it. They will be part of the next pulldown.

Reply:
Thanks for the clarification.